public/sass/components/_gf-form.scss

```diff
@@ -139,8 +147,15 @@ $gf-form-label-margin: 0.25rem;
   }
 }
 
-.gf-form-select-wrapper + .gf-form-select-wrapper { 
-  margin-left: $gf-form-label-margin; 
+.gf-form--textarea {
+  align-items: flex-start;
+}
+
+input[type="number"].gf-natural-language-form {
```

Comment:
worldPing has a type of form style that is in-line, so I wouldnt want to apply it to all gf-form-inputs. That said, I'm sure there is a much better way to do it. Just not positive myself, could use some guidance.